util/settings/settings.go

```diff
@@ -217,6 +217,8 @@ type RepositoryCredentials struct {
 	GithubAppInstallationId int64 `json:"githubAppInstallationID,omitempty"`
 	// Github App Enterprise base url if empty will default to https://api.github.com
 	GithubAppEnterpriseBaseURL string `json:"githubAppEnterpriseBaseUrl,omitempty"`
+	// EnableOCI specifies whether helm-oci support should be enabled for this repo
+	EnableOCI bool `json:"enableOCI,omitempty"`
```

Comment:
Since `RepositoryCredentials` can hold Helm credentials, we need `type` field (`git` or `helm`) as well.